src/lib/dynamodb.ts

```diff
@@ -159,4 +168,8 @@ export class DynamoDB {
 	transactRead(...actions: ReadItem[]) {
 		return new TransactRead(this, actions);
 	}
+
+	batchWrite(items: BatchItem[]) {
```

Comment:
I think it would be better if we had the same syntax as transactions. So instead of passing an array, it's just an argument list.

```suggestion
	batchWrite(...items: BatchItem[]) {
```

Then the example will look like this so make sure to update the documentation.

```ts
const result = await db.batchWrite(
    Table1.createBatchPutItem(
        {partitionKey: 'PK', sortKey: 'SK'},
        {name: 'Sander', lastname: 'Machado'}
    ),
    Table1.createBatchPutItem(
        {partitionKey: 'PK', sortKey: 'SK23'},
        {name: 'Sander', lastname: 'Doe'}
    )
);
```